### ISSUE TITLE
Error when opening gist

### ISSUE THREAD

**Issue by volkerdobler** [vsls-contrib/gistpad#145]
Hello,

on one of my machines (MS Surface with Windows 10 1909 Buildno 18363.778 and VS Code 1.44.2) I can not load my secrete gist.

When I sign in with my accesstoken, gistpad says "Loading gist ..." but no gist is shown.

Looking to the log file, is says:

```
[2020-04-17 17:28:14.476] [exthost] [error] Activating extension vsls-contrib.gistfs failed due to an error:
[2020-04-17 17:28:14.538] [exthost] [error] TypeError: Cannot read property 'uri' of undefined
    at Object.t.isGistWorkspace (c:\Users\volker\.vscode\extensions\vsls-contrib.gistfs-0.0.66\out\prod\extension.js:16:72890)
    at t.activate (c:\Users\volker\.vscode\extensions\vsls-contrib.gistfs-0.0.66\out\prod\extension.js:182:97642)
    at Function._callActivateOptional (c:\Users\volker\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:788:244)
    at Function._callActivate (c:\Users\volker\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:787:909)
    at c:\Users\volker\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:786:960
    at async Promise.all (index 0)
```
Any idea, what I can do to see my gists?

Thanks,
Volker

**Comment by lostintangent** [comment]
@volkerdobler Hmm, this error seems to indicate that you opened up VS Code without a workspace open. Just to double-check: were you using WSL? And if so, did you install GistPad within the WSL environment or "locally"?

**Comment by volkerdobler** [comment]
Hi @lostintangent ,

thanks for your quick answer.
Indeed, I don‘t have a workspace open, just startet VSCode empty. But that is the same on every machine I use and gistpad works on the others fine. Funny enough, if I create a new gist, I can do this, but still, it is not shown on this machine (but the others).

I installed and used Code locally, but the only differences between the machines is, that I played with Remote-WSL extension on this machine (but deinstalled it later). Maybe I just reinstall vscode completely on this machine and start from scratch. I will let you know if this helps.

Regards,
Volker
